EF-Adapter/EF-Adapter.csproj

```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <RootNamespace>Casbin.NET.Adapter.EF</RootNamespace>
+    <PackageId>Casbin.NET.Adapter.EF</PackageId>
+    <Version>1.2.1</Version>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Casbin.NET" Version="1.3.0" />
+    <PackageReference Include="EntityFramework" Version="6.1.3" />
```

Comment:
Why not use the new version like `v6.4.4`?

Reply:
i thought it was in time of implementation the newest but I will update to v.6.4.4